rewrite-groovy/src/main/java/org/openrewrite/groovy/GroovyPrinter.java

```diff
@@ -392,6 +392,16 @@ public J visitForEachLoop(J.ForEachLoop forEachLoop, PrintOutputCapture<P> p) {
             return forEachLoop;
         }
 
+        @Override
```

Comment:
why is this necessary? i believe it extends from `JavaPrinter` and this looks visually similar?